channels/jp.m3u

```diff
@@ -103,3 +101,5 @@ http://stream1.shopch.jp/HLS/out2/prog_index.m3u8
 http://stream1.shopch.jp/HLS/out3/prog_index.m3u8
 #EXTINF:-1 tvg-id="RiBenGouWu4.jp" tvg-name="日本购物4" tvg-country="JP" tvg-language="" tvg-logo="" group-title="",日本购物4
 http://stream1.shopch.jp/HLS/out4/prog_index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV Tokyo
+http://bcsecurelivehls-i.akamaihd.net/hls/live/265320/5043843989001/140130JTDX/index_600.m3u8
```

Comment:
Add this one to have an HD version of this channel : http://bcsecurelivehls-i.akamaihd.net/hls/live/265320/5043843989001/140130JTDX/index_1200.m3u8

Reply:
Already on the playlist.